README.md

```diff
@@ -1,12 +1,12 @@
-# AOZ Voluntar Platform
+# AOZ Voluntaries Platform
```

Comment:
As you only have google translate at hand, and we do not have a project name, I do not see a reason why this should be changed, and then changed to the real name.
Besides: voluntaries is plural of voluntary ;)